src/components/CohortFacilitatorList.tsx

```
@@ -154,11 +159,26 @@ const CohortLearnerList: React.FC<CohortLearnerListProp> = ({
       showToastMessage(t('COMMON.SOMETHING_WENT_WRONG'), 'error');
     }
   }
-  const handleSearch = (searchTerm: string) => {
-    const filtered = userData?.filter((data) =>
-      data?.name?.toLowerCase()?.includes(searchTerm) || data?.enrollmentNumber?.toLowerCase()?.includes(searchTerm)
+  const handleSearch = async (searchTerm: string) => {
+    const filtered = userData?.filter((data) =>{
```

ESLint

[error] 'filtered' is assigned a value but never used.

(no-unused-vars)